### PR TITLE
DOCS-3166 Identify Synthetic Bots Guide Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Removes the deprecated banner from the Browser Variables section and adds a new section for Cookies.

### Motivation

DOCS-3166

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-identify-synthetics-bots-guide-edit/synthetics/guide/identify_synthetics_bots/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 1]
```diff
@@ -99,6 +97,13 @@ if (window._DATADOG_SYNTHETICS_BROWSER === undefined) {
 }
 ```
 
+If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. To avoid this issue, use Edge or Google Chrome.
+
+## Cookies
+
+Cookies that are applied in your browser include `datadog-synthetics-public-id` and `datadog-synthetics-result-id`.
+
+These cookies are available for all steps in Firefox and Internet Explorer 11. For Edge and Google Chrome, these cookies are set only for the start URL.
```
Comment (alai97) [inline content/en/synthetics/guide/identify_synthetics_bots.md:106]:
```suggestion
These cookies are available for all steps in Firefox and Internet Explorer 11. For Edge and Google Chrome, these cookies are set only for the start URL.

```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 2]
```diff
@@ -21,23 +21,25 @@ further_reading:
...
 
 ```
 https://ip-ranges.{{< region-param key="dd_site" >}}/synthetics.json
 ```
 
-**Note:** IPs listed use the CIDR notation and may require conversion to IPv4 address ranges before use.
+The listed IPs use the Classless Inter-Domain Routing (CIDR) notation and may require conversion to IPv4 address ranges before use. Except in the case of new managed locations IPs, the listed IPs rarely change. 
 
-**Note:** Except in the case of new managed locations IPs, the listed IPs rarely change. That being said, if you want to be alerted when the listed IPs change, you can create an API test on the above endpoint with a JSONPath assertion like `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
+That being said, if you want to be alerted when the listed IPs change, create an API test on the endpoint above with a JSONPath assertion such as `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
```
Comment (jtappa) [inline content/en/synthetics/guide/identify_synthetics_bots.md:38]:
```suggestion
If you want to be alerted when the listed IPs change, create an API test on the endpoint above with a JSONPath assertion such as `$.synthetics['prefixes_ipv4_by_location']['aws:ap-northeast-1'].length`.
```

**Review by jtappa — APPROVED** [review]
Just one small suggestion!

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 3]
```diff
@@ -99,6 +97,14 @@ if (window._DATADOG_SYNTHETICS_BROWSER === undefined) {
 }
 ```
 
+If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. To avoid this issue, use Edge or Google Chrome.
```
Comment (peculiarism) [inline content/en/synthetics/guide/identify_synthetics_bots.md:100]: I think we should remove the recommendation to use Edge/Chrome, and instead let customers know they can use a combination of the above methods (maybe at the bottom of the page? Let me know what you think).

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 4]
```diff
@@ -21,23 +21,25 @@ further_reading:
 
 ## Overview
 
-Some parts of your systems might not be available to robots without the right identification, or you might want to avoid collecting analytics from Datadog robots. You can use the below methods to detect Datadog Synthetic Monitoring robots.
+Some parts of your systems may not be available to robots without the appropriate identification. You may also want to avoid collecting analytics from Datadog robots. 
+
+You can use the following methods to detect Datadog's Synthetic Monitoring robots.
```
Comment (peculiarism) [inline content/en/synthetics/guide/identify_synthetics_bots.md:26]: Relating to the comment below, do we want to highlight here that customers can use a combination of the methods for complete detection across Synthetic test configurations?

**Review by peculiarism — CHANGES_REQUESTED** [review]

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 4 above)
Comment (alai97) [reply content/en/synthetics/guide/identify_synthetics_bots.md:26]:
```suggestion
Try a combination of the following methods across your Synthetic test configurations to detect Datadog's Synthetic Monitoring robots.
```

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 3 above)
Comment (alai97) [reply content/en/synthetics/guide/identify_synthetics_bots.md:100]:
```suggestion
If you use the browser variable to identify Synthetic bots on Firefox and Internet Explorer 11, Datadog cannot guarantee that the browser variable is set before your code executes. 
```

File: content/en/synthetics/guide/identify_synthetics_bots.md [hunk 5]
```diff
@@ -21,23 +21,25 @@ further_reading:
 
 ## Overview
 
-Some parts of your systems might not be available to robots without the right identification, or you might want to avoid collecting analytics from Datadog robots. You can use the below methods to detect Datadog Synthetic Monitoring robots.
+Some parts of your systems may not be available to robots without the appropriate identification. You may also want to avoid collecting analytics from Datadog robots. 
+
+Try a combination of the following methods across your Synthetic test configurations to detect Datadog's Synthetic Monitoring robots.
```
Comment (peculiarism) [inline content/en/synthetics/guide/identify_synthetics_bots.md:26]:
```suggestion
You might need to try a combination of the following methods to detect Datadog's Synthetic Monitoring robots.
```

**Review by peculiarism — COMMENTED** [review]
The sentence is more passive, so I'm not sure if we want to change it, let me know what you think.

File: content/en/synthetics/guide/identify_synthetics_bots.md (hunk 5 above)
Comment (alai97) [reply content/en/synthetics/guide/identify_synthetics_bots.md:26]: We use "you may..." in the paragraph right before so using "you might" in this sentence may be a little repetitive IMO.

Comment (alai97) [reply content/en/synthetics/guide/identify_synthetics_bots.md:26]: I would be fine keeping this sentence as is!

**Pull request merged**